Modules/Core/Tests/Feature/UploadControllerTest.php

```
@@ -153,9 +291,34 @@ public function it_rejects_duplicate_file_upload(): void
+        $this->actingAs($user);
+        $response = $this->post(route('upload.upload-file', [
+            'customerId' => 1,
+            'url_key' => '../../../malicious'
+        ]), [
+            'file' => $file,
         ]);
+
+        /** Assert */
+        // Should succeed but sanitize the url_key
+        $response->assertOk();
     }
```

PHPMD (2.15.0)

The method it_sanitizes_url_key_parameter is not named in camelCase. (undefined)

(CamelCaseMethodName)